Homework_24/script.js

```diff
@@ -0,0 +1,72 @@
+
+let a = new Array(13, 22, 31, 4, 5, 16, 7, 8, 9, 10);
+
+
+function addition(a, b) {
+  if (typeof a === "number" || typeof b === "number") {
+    return a + b;
+  } else {
+    console.log("Not a number");
+  }
+}
+
+let division = function (a, b) {
+  if (typeof a === "number" || typeof b === "number") {
+    return a / b;
+  } else {
+    console.log("Not a number");
+  }
+};
+
+const multiplication = (a, b) => {
+  if (typeof a === "number" || typeof b === "number") {
+    return a * b;
+  } else {
+    console.log("Not a number");
+  }
+};
+
+function substraction(a, b) {
+  if (typeof a === "number" || typeof b === "number") {
+    return a - b;
+  } else {
+    console.log("Not a number");
+  }
+}
+
+function min(a) {
+  for (let i = 0; i < a.length; i++) {
+    for (let j = 0; j < a.length - i - 1; j++) {
+      if (a[j] > a[j + 1]) {
+        var temp = a[j];
+        a[j] = a[j + 1];
+        a[j + 1] = temp;
+      }
+    }
+  }
+
+  let min = a[0];
+  return min;
+}
+
+const max = (a) => {
+  for (let i = 0; i < a.length; i++) {
+    for (let j = 0; j < a.length - i - 1; j++) {
+      if (a[j] > a[j + 1]) {
+        var temp = a[j];
+        a[j] = a[j + 1];
+        a[j + 1] = temp;
+      }
+    }
+  }
+  let max = a[9];
```

Comment:
a[9] -> a[a.length - 1]

because if you array will be longer than 9, your code will be broken